.github/workflows/zip-upload.yml

```diff
@@ -0,0 +1,46 @@
+name: Zip and Send
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  zip-and-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
We should be using the reusable action Kari created for then checkout + west update:
https://github.com/NordicBuilder/action-checkout-west-update

Reply:
Done